### DISCUSSION TITLE
solution/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#427] [Announcements]
# solution/

这里是存放题解的位置 Markdown 教学 好用的 markdown 语法 66 我是行内代码块 66 加粗 斜体 上面是分割线 好用的 latex 公式 $a_i,a^i,a\to b$ 网站参考 latexlive $\alpha$ 两侧有两个 $ 是行内居中的 $\iiint\limits{a}^{b}\sum\limits{i=a}^b i$...

https://demooo.top/solution/

**Comment by Demomomomo** [comment]
<h1 align="center">😙</h1>

**Comment by Chivas-Regal** [comment]
<h1 align="center">😙</h1>

**Comment by Meteor-Z** [comment]
佬太强了%%%%%%
